tools/reset_github_user_token_and_update_circleci.sh

```diff
@@ -7,7 +7,6 @@
 #
 # The following env variables must be specified:
 # * USERNAME: a GitHub user to run the script under (Exchange bot).
-# * PASSWORD: password for the user (not a token).
 # * CIRCLECI_TOKEN: a CircleCI token for the Exchange organization.
```

Comment:
Is Github Username and CircleCI token enough now for running this script?
what kind of auth is used?

Reply:
Addressed in the other comment about PASSWORD.